src/test/resources/features/completion/after_pep_object_completion.feature

```diff
@@ -0,0 +1,127 @@
+Feature: Completion after pep object
+
+  Scenario: Complete at toplevel
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    pep.
+    """
+    When I put the caret after pep.
+    And I invoke autocompletion popup
+    Then completion list should contain:
+      | __init__                   |
+      | create_samples             |
+      | _reinit                    |
+      | _get_table_from_samples    |
+      | parse_config_file          |
+      | load_samples               |
+      | modify_samples             |
+      | _modifier_exists           |
+      | attr_remove                |
+      | attr_constants             |
+      | attr_synonyms              |
+      | _assert_samples_have_names |
+      | attr_merge                 |
+      | attr_imply                 |
+      | attr_derive                |
+      | activate_amendments        |
+      | deactivate_amendments      |
+      | add_samples                |
+      | infer_name                 |
+      | get_description            |
+      | __str__                    |
+      | amendments                 |
+      | list_amendments            |
+      | config                     |
+      | config_file                |
+      | samples                    |
+      | sample_name_colname        |
+      | sample_table               |
+      | subsample_table            |
+      | _read_sample_data          |
+      | _get_cfg_v                 |
+      | _format_cfg                |
+      | get_sample                 |
+      | get_samples                |
+
+  Scenario: Complete inside rule
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    rule Name:
+      section:
```

Comment:
`run` section has different parsing structure compared to arg lists section. Please test `run:` also.

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/dcaea2a9a74dff8d433706ce1fff2d61620318b4

Reply:
`onstart` is a toplevel section, not rule section, please update the test

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/424/commits/08c2db8b3c8f83acd9533c6ad152fffd288e51cc